### ISSUE TITLE
PIXI 4.0: DisplayObject.toLocal fails when root object

### ISSUE THREAD

**Issue by Hachitus** [pixijs/pixi.js#2517]
I assume this error shouldn't happen:
https://plnkr.co/edit/MUob9PUb3vOV3g9Ut7pJ?p=preview

So if I call:
```js
var jee = new PIXI.Container();
jee.toLocal(new PIXI.Point(0,0));
```
It will throw an error, because the parent of the Container does not seem to have valid data, it contains pretty much only:
```js
{
    children: Array[0],
    worldAlpha: 1
    worldTransform: Matrix
}
```

Which seems to make sense as, It is initialized only as:
```
_tempDisplayObjectParent = new DisplayObject();
```

So if it is supposed to fail in this case, I think it should give an error when there is no proper parent.

**Comment by Sawamara** [comment]
So this is what I was experiencing when switching to Pixi4.0. 
I had a container which contained a few Graphics object, but toLocal failed as width/height property seems to be undefined on the top level container.

**Comment by Hachitus** [comment]
It's not really a critical braking thing, imo. But it can break v3 -> v4 as it did for me too. So far I've only noticed it have an effect in updateTransform. I did get the error away with if-checks, but that doesn't seem like the correct way to go:
    if (this.parent) {
        this.worldAlpha = this.alpha * this.parent.worldAlpha;
        this.worldProjection = this.parent.worldProjection;
    }
    this.transform.update();
    if (this.parent) {
        this.computedTransform = this.parent.computedTransform.updateChildTransform(this.computedTransform, this.transform);
    }

https://plnkr.co/edit/q0ZqeQjFW3N1MP9JH2T1?p=preview

I do not know the way how PIXI handles these situations, so I didn't come up with a solution I see fir for PR, so it will stay here I guess. I can put if-clauses or make a direct prototype-object as .parent, but I don't think either of those are preferable methods :)
